options/posix/generic/arpa-inet-stubs.cpp

```diff
@@ -65,9 +65,13 @@ in_addr_t inet_addr(const char *) {
 	__ensure(!"Not implemented");
 	__builtin_unreachable();
 }
-char *inet_ntoa(struct in_addr) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
+char *inet_ntoa(struct in_addr addr) {
+	thread_local static char buffer[16];
```

Comment:
Is this buffer big enough? 4*4+1 = 17.

Reply:
four groups of up to three numbers, three periods, and a null: `4 * 3 + 3 + 1 = 16`

Reply:
Add a comment (it's not so hard to see but also not totally trivial).